CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ******
 
+- (`#163 <https://github.com/openclimatedata/openscm/pull/163>`_) Lock ``pyam`` version to pypi versions
```

Comment:
Shouldn't we add a policy to only add bigger changes in the changelog?

Reply:
we can do, I've always just put everything in there but we can do whatever